src/main/java/htsjdk/samtools/filter/TagFilter.java

```diff
@@ -81,7 +108,6 @@ public boolean filterOut(SAMRecord record) {
      */
     @Override
     public boolean filterOut(final SAMRecord first, final SAMRecord second) {
-        // both first and second must have the tag in order for it to be filtered out

```

Comment:
please correct the comment rather than remove it..

Reply:
oh wow nice catch, didn't mean to do that, will remedy